### ISSUE TITLE
Synchronize with contacts option when using Android 5 or lower

### ISSUE THREAD

**Issue by KonomiKitten** [siacs/Conversations#2739]
#### General information

* **Version:** 1.23.3
* **Device:** OPPO R7sf
* **Android Version:** Android 5.1.1 ColorOS 2.1.0i
* **Server name:** jwchat.org
* **Server software:** Unknown
* **Installed server modules:** Unknown
* **Conversations source:** PlayStore

#### Problem

Currently there is no way to enable or disable Synchronize with contacts on Android 5 or lower. As per #1981 it is suggested:

>Android itself has that mechanism build in. If you deny that request two times or something android will allow you to deny it forever.

And this would be true of Android 6 and up but this is not true for people on Android 5 or lower. I am therefor requesting an actual option in the client to disable this feature as a lot of privacy minded people find this to be a bit too much even if it's just reading contacts and matching them up to your list, people might not even want that to happen for aesthetic reasons anyway.

Thanks for your time.

**Comment by licaon-kter** [comment]
You know that the entries are NOT sent on the internet, right? They are not "matched" to any other list.

**Comment by KonomiKitten** [comment]
> @licaon-kter 
> You know that the entries are NOT sent on the internet, right? They are not "matched" to any other list.

Yes I am, hence why I stated:

> @KonomiKitten
> ...a lot of privacy minded people find this to be a bit too much even if it's just reading contacts and matching them up to your list, people might not even want that to happen for aesthetic reasons anyway.

**Comment by licaon-kter** [comment]
Maybe rephrase `matching up to *your* list` to `matching up with *their own* list`, else it means something else.

**Comment by iNPUTmice** [comment]
Just don’t put the Jabber IDs in the android address book. Or use some external tool to revoke the dependencies. Conversations will behave just fine when you do.
